ember/app/components/project-form/component.js

```diff
@@ -0,0 +1,101 @@
+import { action } from '@ember/object';
+import { scheduleOnce } from '@ember/runloop';
+import { service } from '@ember/service';
+import Component from '@glimmer/component';
+import emptyChangeset from 'outdated/utils/empty-changeset';
+import ProjectValidations from 'outdated/validations/project';
+import { tracked } from 'tracked-built-ins';
+
+export default class ProjectFormComponent extends Component {
+  // Services
+  @service store;
+  @service router;
+  @service notification;
+
+  @tracked project = emptyChangeset(
+    ProjectValidations,
+    this.args.project ?? this.store.createRecord('project')
+  );
+  @tracked editing = !!this.args.project;
+
+  constructor(...args) {
+    super(...args);
+    if (this.args.project) {
+      scheduleOnce('actions', this, 'initUsers');
+    }
+  }
+
+  initUsers() {
+    this.project.users = this.users.filter((u) =>
+      this.maintainers
+        .map((maintainer) => maintainer.user.get('id'))
+        .includes(u.id)
+    );
+    this.project.primaryMaintainer =
+      this.project.users.find(
+        (u) =>
+          u.id ===
+          this.maintainers
+            .find((maintainer) => maintainer.isPrimary)
+            ?.user?.get('id')
+      ) ?? this.project.users[0];
+  }
+
+  @action
+  async saveProject() {
+    try {
+      const project = await this.project.save();
+      this.maintainers.forEach((maintainer) => {
+        if (
+          !this.project.users
+            .map((u) => u.id)
+            .includes(maintainer.user.get('id'))
+        ) {
+          maintainer.destroyRecord();
+        } else {
+          maintainer.isPrimary =
+            maintainer.user.get('id') === this.primaryMaintainer.id;
+          maintainer.save();
+        }
+      });
+
+      this.project.users.forEach((user) => {
+        if (!this.maintainers.map((m) => m.user.get('id')).includes(user.id)) {
+          this.store
+            .createRecord('maintainer', {
+              user,
+              project,
+              isPrimary: user === this.primaryMaintainer,
+            })
+            .save();
+        }
+      });
+
+      this.router.transitionTo('projects.detailed', project.id);
+      this.notification.success('Successfully saved!', {
+        pos: 'bottom-right',
```

Comment:
Why is this differently positioned than the other notifications?

Reply:
Its positioned the same as the other notifications, when i globally defined the position (https://github.com/adfinis/Outdated/pull/14/commits/805c733a5f1cbcef9d419eefc5f57650db2bd6b9) I missed this one. Fixed it here: https://github.com/adfinis/Outdated/commit/c965aee52e37ef22d4143f91c193b9f67274973a#diff-b2fa88cc6fbb01d460606fa429ca33de8fadb4914a6c0e04b60fd8c6297fb999R75